crates/compute/src/layer.rs

```diff
@@ -299,8 +301,7 @@ pub trait ComputeLayer<F: Field>: 'static {
 	fn sum_composition_evals(
 		&self,
 		exec: &mut Self::KernelExec,
-		log_len: usize,
-		inputs: &[FSlice<'_, F, Self>],
+		inputs: &SlicesBatch<FSlice<'_, F, Self>>,
```

Comment:
@BenjaminTrapani @iradicek Please note the interface change here

Reply:
I say this because earlier this was a slice of column-slices and now `SlicesBatch` takes `rows`

Reply:
> Hm, it seems we cannot access individual column anymore.

@iradicek Actually we can via `SlicesBatch::row` method. The major difference between `&[FSlice<'_, F, Self>]` and `&SlicesBatch<FSlice<'_, F, Self>>` is that for the latter we don't need to check that all slices are of the same length inside the function which may save quite a signifficant time when we re-use the same big batch in many calls each of them accessing only a a few rows (that is a real case we met in CPU implementations)